src/Uno.UWP/FeatureConfiguration/WinRTFeatureConfiguration.FileTypes.cs

```diff
@@ -22,27 +22,28 @@ public static class Storage
 		{
 			public static class Pickers
 			{
-#if UNO_REFERENCE_API
+#if __CROSSRUNTIME__
```

Comment:
Both `__CROSSRUNTIME__` and `UNO_REFERENCE_API` are currently true for Wasm, Skia, and Reference.

Does this change have no impact, but for future when UNO_REFERENCE_API is properly fixed to mean reference only?

Reply:
I mainly changed this for consistency, the `__CROSSRUNTIME__` seems clearer here